### ISSUE TITLE
[Grid] Using a Grid with column configuration within cell template causes an error

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#2187]
When another Grid is nested within a parent Grid Cell template, and the columns configuration is present in the markup, the following error is thrown in development mode:

ERROR Error: Columns can be nested only inside ColumnGroupComponent

[https://stackblitz.com/edit/angular-hlecdm-zdz25j?file=app/app.component.ts](https://stackblitz.com/edit/angular-hlecdm-zdz25j?file=app/app.component.ts)

When the columns are autogenerated, the nested Grid is rendered as expected.

**Comment by v-srsi** [comment]
hi,
any workaround on this?

**Comment by tsvetomir** [comment]
The recommended workaround is to encapsulate the nested grid in a component, [see demo](https://stackblitz.com/edit/angular-hlecdm-1ebxmf?file=app/app.component.ts):

```html
<kendo-grid-column field="innerGridData" width="300">
  <ng-template kendoGridCellTemplate let-dataItem>
    <app-nested-grid [data]="dataItem.innerGridData"></app-nested-grid>
  </ng-template>
</kendo-grid-column>
```

**Comment by Raisolution** [comment]
The issue has been addressed in `v5.5.1` of the `kendo-angular-grid` package.
